<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page_end</class>
 <widget class="QWidget" name="page_end">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>1920</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_to_idle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>1080</width>
     <height>1920</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(/home/df-admin/production/references/7_background_thank_you.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_message">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>590</y>
     <width>841</width>
     <height>471</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_customer_logo">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>1710</y>
     <width>351</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_message_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>1050</y>
     <width>1081</width>
     <height>511</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_manufacturer_logo">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>471</width>
     <height>331</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_contact">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>1404</y>
     <width>380</width>
     <height>101</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777214</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="toolTipDuration">
    <number>-10</number>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../production/references/helpButton.png</normaloff>../../production/references/helpButton.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_volume_dispensed">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>1530</y>
     <width>600</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Volume dispensed</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_volume_dispensed_ml">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>1620</y>
     <width>600</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
